helpers/any.js

```diff
@@ -24,22 +22,24 @@ const factory = () => {
         opts = args.pop();
         predicate = opts.hash;
 
-        if (!_.isEmpty(predicate)) {
+        if (Object.keys(predicate).length !== 0) {
             // With options hash, we check the contents of first argument
-            any = _.any(args[0], predicate);
+            any = args[0].some(arg => {
```

Comment:
`args[0]` could be either an object or an array, but we are assuming it's an array here. The lodash implementation supports both: https://github.com/lodash/lodash/blob/3.10.1/lodash.js#L7044

Reply:
Here's some documentation on this helper to help produce a correct implementation: https://developer.bigcommerce.com/stencil-docs/handlebars-syntax-and-helpers/handlebars-helpers-reference/new-subpage/loop-control-flow-helpers#handlebars_any

Reply:
ok will change.